docs/features/commands.md

```diff
@@ -45,4 +45,23 @@ Add a member to either a specific ticket or the ticket channel the command is us
 	/add member: @eartharoid ticket: #ticket-91
 	```
 
-### `/claim`
\ No newline at end of file
+### `/priority`
+
+Sets an emoji in front of the channel name to declare the tickets priority
+
+#### Options
+
+|                          Name                            |                            Description                            |    Example    |
+| :------------------------------------------------------: | :---------------------------------------------------------------: | :-----------: |
+| :material-asterisk:{ .pink title="Required" } `priority` |                  The priority to set		               |     `HIGH`    |
+
+??? example
```

Comment:
I don't think examples are necessary for this command because there is a single required option, and it has choices to choose from rather than being free input.